.rhcicd/clowdapp.yaml

```diff
@@ -140,6 +140,41 @@ objects:
                         secretKeyRef:
                           name: policies-ui-backend-db
                           key: ${DB_SECRET_PASSWORD_KEY}
+  - apiVersion: metrics.console.redhat.com/v1alpha1
+    kind: FloorPlan
+    metadata:
+      name: policies-backend-hms
+    spec:
+      database:
+        secretName: ${FLOORIST_DB_SECRET_NAME}
+      objectStore:
+        secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+      logLevel: ${FLOORIST_LOGLEVEL}
+      suspend: ${{FLOORIST_SUSPEND}}
+      disabled: ${{FLOORIST_DISABLED}}
+      queries:
+        - prefix: hms_analytics/policies/status
+          query: >-
+            SELECT
+              id,
+              name,
+              org_id,
+              is_enabled,
+              conditions,
+              mtime,
+              ctime,
+              actions,
+              JSON_AGG(policy_history.host_id) as hosts
+            FROM
+              policy
+            JOIN
+              policy_history
+            ON
+              policy_history.policy_id = policy.id
+            GROUP BY
+              id;
```

Comment:
This is not enough for the JSON_AGG. 
```
Error: ERROR: column "policy.name" must appear in the GROUP BY clause or be used in an aggregate function (SQLSTATE 42803)
```
It would likely need a list of all fields on the group by.